### PR TITLE
PHPStan level 6 green

### PR DESCRIPTION
**Pull request by dereuromark**
CS found also some invalid newlines (windows ones)? and phpcs fixed it

### CODE REVIEW

**Comment by dereuromark** [comment]
whats up with this phive thing?
If it isnt so stable working, maybe a manual install might be easier after all?

**Comment by dereuromark** [comment]
@burzum Can we get this merged please?

I would like to check for 1.0 stable release what might be missing so far. We need this for the cake plugin to become stable ideally.

**Comment by floriankraemer** [comment]
CI seems to fail? Can you please fix it.

> whats up with this phive thing?

The one problem that I had with it was fixed month ago. What issue do you encounter?

File: phive.xml [hunk 1]
```diff
@@ -1,35 +0,0 @@
-<?xml version="1.0"?>
```
Comment (floriankraemer) [inline phive.xml]: Please revert that file.

**Comment by dereuromark** [comment]
This is the old location 
The tool says it only wants the new location

**Comment by dereuromark** [comment]
Ci issues are not on me. Locally it works fine

**Comment by floriankraemer** [comment]
> Ci issues are not on me. Locally it works fine

Well, then you'll have to wait until I have time and mood to fix it.

File: phive.xml (hunk 1 above)
Comment (dereuromark) [reply phive.xml]: Oh, it didnt commit the new file location it was suggesting in its latest version
Fixed now

**Comment by dereuromark** [comment]
Hah, now it makes more sense.
Seems like all is fine now.

The remaining error seems to be resolved with https://github.com/Phauthentic/file-storage-image-processor/pull/9/files

**Comment by dereuromark** [comment]
Can we release an RC maybe?

**Pull request merged**